ngraph/test/backend/gather_elements.in.cpp

```diff
@@ -368,15 +368,15 @@ NGRAPH_TEST(${BACKEND_NAME}, evaluate_2D_gather_elements_2x2x1_data_float32)
 
     test_case.add_input<int32_t>(data);
     test_case.add_input<int32_t>(indices);
-    test_case.add_expected_output<float>(vector<float>{5,
-                                                       4,
-                                                       1,
-                                                       4,
-
-                                                       1,
-                                                       4,
-                                                       5,
-                                                       4});
+    test_case.add_expected_output<int32_t>({5,
```

Comment:
Did this change from float to int32 help to enable several tests Or do we also have some changes on plugins side?

Reply:
This change fixes wrong output type - it has to be the same as input node data, which is int32 in this test (arg1 parameter and data vector definitions).